admin/app/controllers/authenticated/certifications/certification/details.js

```diff
@@ -0,0 +1,95 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { alias } from '@ember/object/computed';
+import Controller from '@ember/controller';
+import { schedule } from '@ember/runloop';
+
+export default class CertificationDetailsController extends Controller {
+  // Properties
+  juryRate = false;
+
+  juryScore = false;
+  requestedId = '';
+
+  // Private properties
+  @service('mark-store')
+  _markStore;
+
+  // Aliases
+  @alias('details.percentageCorrectAnswers')
+  rate;
+
+  @alias('details.totalScore')
+  score;
+
+  @alias('model')
+  details;
+
+  @action
+  onUpdateRate() {
+    const competences = this.get('details.competences');
+    const { good, count, jury } = _getCertificationResultsAfterJuryUpdate(competences);
+
+    if (jury) {
+      this.set('juryRate', Math.round(good * 10000 / count) / 100);
```

Comment:
```suggestion
      this.juryRate = Math.round(good * 10000 / count) / 100);
```

Reply:
Tu peux corriger dans tout le fichier

Reply:
Pour ce fichier là j'ai pas réussi à octaniser à cause du "schedule" , j'ai l'impression que ça met le bazarre dans le contexte du "this" ... Et j'ai l'impression que mélanger les syntaxes ne fonctionne pas

Reply:
Je crois que c'est pour ça aussi que @jbuget n'avait pas retouché ce fichier lorsqu'il a upgrade la version de ember